presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1105,6 +1106,27 @@ public Void visitApply(ApplyNode node, Integer indent)
         }
 
         @Override
+        public Void visitJoinGraph(JoinGraphNode node, Integer indent)
+        {
+            List<Expression> joinExpressions = new ArrayList<>();
```

Comment:
the other visitors in this class seem to use regular lists, so I was going for consistency  I can change this anyway if you'd like.